### PR TITLE
Privacy Policy related copy updates

### PR DESCRIPTION
**Pull request by moollaza**
## Description

- Updates several of our privacy policy "one-liners"
- Also converted some HTML to MD

### CODE REVIEW

File: _docs/features/autosuggest.md [hunk 1]
```diff
@@ -4,18 +4,8 @@ category: Features
...
-    To be clear,
-    <a href="https://duckduckgo.com/privacy">we don't collect or share personal information and autosuggest does not
-        impact that at all</a>. We simply do not associate queries with personal information (e.g. IP
-    addresses) and in fact don't store any of it at all so our autosuggest
-    doesn't impact your privacy!
-</p>
+Autosuggest (also known as autocomplete) is available for DuckDuckGo across platforms, including [our website](https://duckduckgo.com/), [browser extensions](/desktop/adding-duckduckgo-to-your-browser), [Android app](/mobile/android) and [iOS app](/mobile/ios).
 
-<img src="{{ site.baseurl }}/images/31b134d0c227539ea4a23b6e154b85d7.png" />
+To be clear, [we don't track you, and autosuggest does not impact that at all](https://duckduckgo.com/privacy) — we simply do not associate autocomplete queries with unique identifiers (e.g., IP addresses) so that no search histories can be created.
```
Comment (moollaza) [inline _docs/features/autosuggest.md:9]: Updated copy here

File: _docs/privacy/atb.md [hunk 2]
```diff
@@ -4,79 +4,22 @@ category: Privacy
...
 ---
 
-<p>
-    At DuckDuckGo, we do not collect or share any personal information. That's our
-    <a href="https://duckduckgo.com/privacy">privacy policy</a> in a nutshell. For
-    example, we do not store IP addresses, and we do not create unique cookies. As
-    such, we do not even have the ability to create search histories or search
-    sessions for any individual — privacy by design.
-</p>
+At DuckDuckGo, we don’t track you, ever. That’s our [privacy policy](https://duckduckgo.com/privacy) in a nutshell. For example, we do not create unique cookies and, more generally, architect our product so that we do not even have the ability to create a search or browsing history for any individual — it’s privacy by design.
```
Comment (moollaza) [inline _docs/privacy/atb.md:7]: Updated copy here

File: _docs/privacy/no-tracking.md [hunk 3]
```diff
@@ -4,16 +4,4 @@ category: Privacy
...
-    also offer two non-JavaScript versions (<a href="https://duckduckgo.com/html">HTML</a>
-    &amp; <a href="https://duckduckgo.com/lite">lite</a>), a
-    <a href="https://duckduckgogg42xjoc72x3sjasowoarfbgcmvfimaftt6twagswzczad.onion/">Tor onion service</a>, several
-    <a href="https://duckduckgo.com/settings">privacy settings</a> (including POST
-    &amp; RefControl), and we allow you to use
-    <a href="https://duckduckgo.com/params">URL parameters</a> instead of cookies
-    for our Cloud Save feature to store settings.
-</p>
+DuckDuckGo never tracks you, per [our privacy policy](https://duckduckgo.com/privacy). In addition to [our default version](https://duckduckgo.com/), we also offer two non-JavaScript versions ([HTML](https://duckduckgo.com/html) & [lite](https://duckduckgo.com/lite)), a [Tor onion service](https://duckduckgogg42xjoc72x3sjasowoarfbgcmvfimaftt6twagswzczad.onion/), several [privacy settings](https://duckduckgo.com/settings) (including POST & RefControl), and we allow you to use [URL parameters](https://duckduckgo.com/params) instead of cookies for our Cloud Save feature to store settings.
```
Comment (moollaza) [inline _docs/privacy/no-tracking.md:7]: Updated copy here

File: _docs/privacy/smarter-encryption.md [hunk 4]
```diff
@@ -1,83 +1,28 @@
...
 ---
 
-<p>
-    At DuckDuckGo, we do not collect or share any personal information. That's our
-    <a href="https://duckduckgo.com/privacy">privacy policy</a> in a nutshell. For
-    example, we do not store IP addresses in our server logs, and we do not create
-    unique cookie IDs. As such, we do not even have the ability to create search or
-    browsing histories for any individual — it's privacy by design.
-</p>
+At DuckDuckGo, we don’t track you, ever. That’s our [privacy policy](https://duckduckgo.com/privacy) in a nutshell. For example, we do not create unique cookies and, more generally, architect our product so that we do not even have the ability to create a search or browsing history for any individual — it’s privacy by design.
```
Comment (moollaza) [inline _docs/privacy/smarter-encryption.md:7]: Updated copy here

File: _docs/results/sources.md [hunk 5]
```diff
@@ -4,36 +4,10 @@ category: Results
...
-personal information with any of our partners. The way it works is when we call
-a partner for information, it is proxied through our servers so it stays
-completely anonymous. That is, any call to a partner looks to the partner as it
-is from us and not the user itself, and no user personal information is passed
-in that process (e.g. their IP address). That way we can build our search result
-pages using these 100s of partner sources, while still keeping them completely
-anonymous to you.
+Partners and Privacy: As per our strict [privacy policy](https://duckduckgo.com/privacy), we never share any personal information with any of our partners that could lead to the creation of search histories. The way it works is when we call a partner for information used in search results, it is proxied through our servers so it stays anonymous. That is, any call to a partner looks to the partner as it is from us and not the user itself, and no unique identifier is passed in that process (e.g., their IP address). That way we can build our search result pages using these 100s of partner sources, while still keeping them anonymous to you.
```
Comment (moollaza) [inline _docs/results/sources.md:13]: Updated copy here

File: _docs/privacy/no-tracking.md (hunk 3 above)
Comment (tagawa) [reply _docs/privacy/no-tracking.md:7]: One tiny suggestion – could say "our anonymous Cloud Save feature" here, to clarify that it's not connected with an email address or other personal info.

File: _docs/results/sources.md (hunk 5 above)
Comment (tagawa) [reply _docs/results/sources.md:13]: Could possibly remove "The way it works is" here, to make the copy shorter while keep the meaning.

Also, seems to be missing a "though":
> looks to the partner as though it is from us

File: _docs/privacy/no-tracking.md (hunk 3 above)
Comment (moollaza) [reply _docs/privacy/no-tracking.md:7]: Thanks @tagawa, I'll pass this along to the Comms team

File: _docs/privacy/privacy-labels.md [hunk 6]
```diff
@@ -6,22 +6,24 @@ order: 80
...
 
-You can use the DuckDuckGo app knowing that no personal data is being collected by the app itself, or via its features (like DuckDuckGo Search), and that we are also helping protect your data on the websites you visit (through our [Tracker Radar](https://spreadprivacy.com/duckduckgo-tracker-radar/) and [Smarter Encryption](https://spreadprivacy.com/duckduckgo-smarter-encryption/) technologies).
+You can use the DuckDuckGo app knowing that the app itself, or its features (like DuckDuckGo Search), aren't tracking you and that we are also helping protect your data on the websites you visit (e.g., through our [Tracker Radar](https://spreadprivacy.com/duckduckgo-tracker-radar/) and [Smarter Encryption](https://spreadprivacy.com/duckduckgo-smarter-encryption/) technologies) technologies).
```
Comment (ogasquez) [inline _docs/privacy/privacy-labels.md:15]: There is duplication here: `technologies) technologies).`

**Pull request merged**